docs/support_playbook.md

```diff
@@ -98,62 +98,56 @@ Make sure you know which part you are amending. Add `\r\n\r\n` for carriage retu
 ApplicationForm.find(_id).update!(becoming_a_teacher: 'new text', subject_knowledge: 'new_text', audit_comment: 'Updating grade following a support request, ticket ZENDESK-LINK')
 ```
 
-## Offers
-
-### Make or change offer
-
-If the current application status is `awaiting_provider_decision` use MakeAnOffer service.
+## Courses and course locations 
 
-If the current application status is `offer` use ChangeOffer service.
-
-### Change provider/course
-
-Sometimes a provider has no room for an accepted candidate and they refer them to another provider. In this case we create a new ApplicationChoice for the new course_option by following these steps:
+### Changing a course or course location 
 
-- Get the old application choices to withdraw later
+A provider may request that a candidate be placed on an different course, or a different site. 
 
-For example by candidate’s email:
+- Find the provider 
 
 ```ruby
-old_application_choice = ApplicationForm.find_by(candidate_id: Candidate.find_by(email_address: 'example@email.com').id).application_choices.first
+provider = Provider.find_by(code: _provider_code)
 ```
 
-- Add course after submission
-
-Find the course option for the new course. As the course code is not unique make sure to include the provider details.
+- Find the new course you wish to change to (in the event you are simply updating the site you will need to find the site you wish to change to)
 
 ```ruby
-course_option = CourseOption
-  .where(
-    course_id: Course.where(code: _course_code, provider: Provider.find_by(code: _provider_code)),
-    recruitment_cycle_year: RecruitmentCycle.current_year
-  )
+new_course = Course.where(
+  code: _course_code, 
+  provider: provider, 
+  recruitment_cycle_year: RecruitmentCycle.current_year
 ).first
 ```
 
-- Find the application form
-
-Eg -
+- Find the associated course option 
 
 ```ruby
-application_form = old_application_choice.application_form
+new_course_option = CourseOption.where(course_id: new_course.id).first
```

Comment:
A course has multiple course options assigned to it, so this is not quite right as we are randomly just picking the first one.

Reply:
yes — we should depend on a `CourseOption`, not a `Course`.